### DISCUSSION TITLE
Issue with vercel template

### DISCUSSION THREAD

**Discussion by jkabat** [wevm/frog-archived#520] [Q&A] [answered]
I'm new to the project so this is going to be a lame newbie question.

After installing a vercel project scaffold and running `npm frog dev` I'm getting following error:

```bash
Error: Missing "./vercel" specifier in "frog" package
```

I have globally installed vercel and linked the project. What am I missing here?

**Comment by dalechyn** [comment, marked as answer]
> I'm new to the project so this is going to be a lame newbie question.
> 
> After installing a vercel project scaffold and running `npm frog dev` I'm getting following error:
> 
> ```shell
> Error: Missing "./vercel" specifier in "frog" package
> ```
> 
> I have globally installed vercel and linked the project. What am I missing here?

Hey! It is frog's issue, you have to change import from `frog/vercel` to `frog/next`.
It seems like templates were not updated
